src/components/Review/ReviewComment.tsx

```diff
@@ -44,6 +45,7 @@ const ReviewComment: React.FC<ReviewCommentProps> = ({ reviewDecisionId, isEdita
     <div>
       <p>
         <strong>{strings.LABEL_REVIEW_OVERALL_COMMENT}:</strong>
+        <Tooltip message={strings.LABEL_REVIEW_OVERALL_COMMENT_TOOLTIP} />
```

Comment:
I think setting a max-width of around 500 looks better for this one in particular:
![Screen Shot 2022-08-12 at 2 59 43 PM](https://user-images.githubusercontent.com/5456533/184276956-4c86797f-4290-47b8-9baf-c4f9ef721061.png)

Probably should add a few basic style props to this component cos different cases would want different widths.

You probably want at least:
- min-width
- max-width

Make sure they have defaults though (probably 500px for max-width, and just let the Popup component use its own min-width)